cocotb/clock.py

```diff
@@ -30,10 +30,11 @@
 import os
 import itertools
 
-if "SPHINX_BUILD" in os.environ:
-    simulator = None
-else:
+if "COCOTB_SIM" in os.environ:
     import simulator
+else:
+    simulator = None
```

Comment:
Might be worth putting this in its own PR and merging it separately - looks obviously worthwhile.

Reply:
Right!

Reply:
Yeah, seems this was forgotten when doing the conversion.

Reply:
Is the simulator even needed in this file?

Reply:
Seems not ;-)